dist/disemvowling.js

```diff
@@ -0,0 +1,12 @@
+const readlineSync = require('readline-sync');
+
+const stringString = readlineSync.question('Write a funny sentence\n');
+const stringArray = stringString.split('');
+const goAwayCharacters = /[aeiou|\s]/;
+for (let index = 0; index < stringArray.length; index++) {
+  if (goAwayCharacters.test(stringArray[index])) {
+    stringArray.splice(index, 1);
+    index--;
+  }
+}
+console.log(newString = stringArray.join(''));
```

Comment:
This only filters out lowercase vowels. You can either add the uppercase vowels to the regex that is filtered out or you can `.toLowercase()` the string before it get's filtered by the `goAwayCharacters` function. Otherwise solid logic and code.